fluent-package/apt/systemd-test/update-from-v4-with-old-service-file.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash
+
+set -exu
+
+. $(dirname $0)/../commonvar.sh
+
+# TODO: Remove it when v5 repository was deployed
+sudo apt install -y curl ca-certificates
+curl -O https://packages.treasuredata.com/4/${distribution}/${code_name}/pool/contrib/f/fluentd-apt-source/fluentd-apt-source_2020.8.25-1_all.deb
+sudo apt install -y ./fluentd-apt-source_2020.8.25-1_all.deb
+
+# Install v4
+sudo apt clean all
+# Uncomment when v5 repository was deployed
+#apt_source_package=/vagrant/${distribution}/pool/${code_name}/${channel}/*/*/fluent-apt-source*_all.deb
+#sudo apt install -V -y ${apt_source_package} ca-certificates
+sudo apt update
+sudo apt install -V -y td-agent=4.5.0-1
+
+systemctl status --no-pager td-agent
+
+# Set custom service as "td-agent-server"
+sudo systemctl stop td-agent
+sudo cp /lib/systemd/system/td-agent.service /etc/systemd/system/td-agent-server.service
+sudo cp /etc/logrotate.d/td-agent /etc/logrotate.d/td-agent-server
+sudo cp /etc/td-agent/td-agent.conf /etc/td-agent/td-agent-server.conf
+sudo sed -i /etc/systemd/system/td-agent-server.service -e 's /etc/td-agent/td-agent.conf /etc/td-agent/td-agent-server.conf g'
+sudo sed -i /etc/systemd/system/td-agent-server.service -e 's /var/run/td-agent/td-agent.sock /var/run/td-agent-server/td-agent-server.sock g'
```

Comment:
```suggestion
sudo sed -i /etc/systemd/system/td-agent-server.service -e 's,/var/run/td-agent/td-agent.sock,/var/run/td-agent-server/td-agent-server.sock,g'
```

Could you explicitly set non-space separator?
It must be exactly 1 space, but it may be bit hard to distinguish. ` `  `  `

Reply:
I see! Thanks for the advice!
I will fix it.

Reply:
Fixed.